### PR TITLE
Publish 6.0.2 milestone

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Publish 6.0.2 milestone

### Changed

-   Added 1rem default padding to `<blui-empty-state>` ([#336](https://github.com/brightlayer-ui/angular-component-library/issues/336)).
-   Added a list `peerDependencies` to component's `package.json` ([#351](https://github.com/brightlayer-ui/angular-component-library/issues/351)). 

### Fixed

-   Fixed `<blui-app-bar>` not listening for new scroll elements `onChanges` ([#362](https://github.com/brightlayer-ui/angular-component-library/issues/362)).
-   Fixed `<blui-user-menu>` not rendering bottomsheet content when quickly dismissing and then reopening. ([#345](https://github.com/brightlayer-ui/angular-component-library/issues/345))
-   Fixed `<blui-drawer-nav-item>` not responding to `expanded` input updates. ([#326](https://github.com/brightlayer-ui/angular-component-library/issues/326))
-   Fixed `<blui-drawer-nav-item>` not applying correct `depth` class for asynchronously loaded items. ([#356](https://github.com/brightlayer-ui/angular-component-library/issues/356))

### CODE REVIEW

**Comment by emclaug2** [comment]
I'm going to convert this to a draft; there's still some items in here that should be demoed before publishing.

File: components/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@brightlayer-ui/angular-components",
-    "version": "6.0.1",
+    "version": "6.0.2-beta.3",
```
Comment (emclaug2) [inline components/package.json:3]: This should be updated.

**Comment by emclaug2** [comment]
These bug fixes/features have been demo'd this morning, converting this to a draft.

**Review by huayunh — APPROVED** [review]

**Review by jeffvg — APPROVED** [review]
looks good

**Pull request merged**